tools/autotag/templates/rocm_changes/5.6.1.md

```diff
@@ -0,0 +1,20 @@
+<!-- markdownlint-disable first-line-h1 -->
+<!-- markdownlint-disable no-duplicate-header -->
+
+### What's New in This Release
+
```

Comment:
ROCm 5.6.1 is a point release with several bug fixes in the HIP runtime.

Reply:
added in https://github.com/RadeonOpenCompute/ROCm/pull/2416